ddtrace/internal/core/__init__.py

```diff
@@ -163,23 +163,24 @@ def _deprecate_span_kwarg(span):
         )
 
 
-class ExecutionContext:
-    __slots__ = ["identifier", "_data", "_parents", "_span", "_token"]
```

Comment:
Why's this removed?

Reply:
Because now ExecutionContext is a real context, before it was just a data class inside a context. And sadly, support for slots in python context was only [merged in main cpython branch this june ](https://github.com/python/cpython/issues/105726)
Technically it's more efficient that way, because we remove the intermediary context item. Less memory and no noticeable time difference.